queries/ecma/highlights.scm

```diff
@@ -48,6 +103,9 @@
   name: (identifier) @function)
 (method_definition
   name: [(property_identifier) (private_property_identifier)] @method)
+(method_definition
+  name: (property_identifier) @method.builtin
+  (#eq? @method.builtin "constructor"))
```

Comment:
why not `@constructor` 🤨

Reply:
It's not really a constructor. :thinking: